### ISSUE TITLE
browser support - clarify default browserlist configuration differences with documentation

### ISSUE THREAD

**Issue by bbarry** [angular/angular#37744]
# 📚 Docs or angular.io bug report

### Description

 
https://angular.io/guide/browser-support

Specifically, this page mentions support for IE 11 but the default browserlist for new applications is:

```
last 1 Chrome version
last 1 Firefox version
last 2 Edge major versions
last 2 Safari major version
last 2 iOS major versions
Firefox ESR
not IE 9-11 # For IE 9-11 support, remove 'not'.
```

which leaves out IE11. 

Perhaps this page should include the default browser list file for new applications and comment why this is the default if more than this is actually supported?

### What's the affected URL?**
 https://angular.io/guide/browser-support

### Reproduction Steps**

 

1. visit https://angular.io/guide/browser-support
2. read https://blog.angular.io/version-10-of-angular-now-available-78960babd41?gi=8bca91b3153b and note the stuff it says about ES5 and realize IE 11 is not added automatically then wonder what the default browserlist file actually is. 
3. realize there is no easy documented way to see the default list
4. `ng new temp` and cancel it before it finishes installing everything but after it creates files (or just let it work if it is fast enough)
5. view browserlist

### Expected vs Actual Behavior**

Expecting to see one of:

* IE 11 deprecated officially
* IE 11 deprecated for new applications
* note about IE 11 support status vs default configuration

Expecting to see:

* documentation about the default browserlist file and reasoning for why it is the way it is (like: why doesn't angular recommend `default`?)

**Comment by IgorMinar** [comment]
yeah. I agree that we could have split the the IE entries into two lines with better comments. for example:

```
last 1 Chrome version
last 1 Firefox version
last 2 Edge major versions
last 2 Safari major version
last 2 iOS major versions
Firefox ESR
not IE 9-10 # Angular support for IE 9 & 10 has been deprecated and will be removed as of Angular v10.
not IE 11 # Angular supports IE 11 only as an opt-in. To opt-in, remove the `not` prefix on this line.
```

@alan-agius4 @clydin what do you think?
